### PR TITLE
Simple money calculator

### PR DESCRIPTION
**Pull request by flackovic**
Simple money calculator. Works with ints and floats.
It is expected to have 2 decimal precision

### CODE REVIEW

File: src/Calculator.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    {
+        return $a - $b;
+    }
+
+    public function multiply($a, $b)
+    {
+        return $a * $b;
+    }
+
+    public function calculateWithPercentage($input, $percentage)
```
Comment (HcsOmot) [inline src/Calculator.php]: In Php >= 7 we can use `float` type hints and compiler will automatically convert `int` values to `float`.

File: src/Calculator.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    {
+
+        if(is_float($input)) {
```
Comment (HcsOmot) [inline src/Calculator.php]: That way you don't have to manually check for type.

File: tests/CalculatorTest.php [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+    public function test100PercentsWillDoubleTheInput()
+    {
+        $result = $this->calculator->calculateWithPercentage(10, 100);
+
+        self::assertEquals(20, $result);
+    }
+
+
+    public function test25PercentsWillWork()
```
Comment (HcsOmot) [inline tests/CalculatorTest.php]: Perhaps consider renaming this method to something more descriptive (25 percent will work what/how/when?)

File: tests/CalculatorTest.php [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+    public function test25PercentsWillRoundUp()
+    {
+        $result = $this->calculator->calculateWithPercentage(10001, 25);
+
+        self::assertEquals(12502, $result);
+    }
+
+
+    public function testFloatsWillWork()
```
Comment (franraknic) [inline tests/CalculatorTest.php]: za malo više razumljivosti, budući da testiramo povratni tip moguće je:
`
self::assertInternalType(PHPUnit_IsType::TYPE_FLOAT, $result);
`
ili
`
self::assertInternalType("float", $id);
`

**Pull request merged**